jest.config.js

```diff
@@ -19,7 +29,9 @@ module.exports = {
     // `graceful-fs` does `require('fs')`.
     // Ref: https://github.com/facebook/jest/issues/2179#issuecomment-355231418
     // If this is removed, see also scripts/build/build.js.
-    "graceful-fs": "<rootDir>/tests_config/fs.js"
+    "graceful-fs": "<rootDir>/tests_config/fs.js",
+
+    "prettier/local": "<rootDir>/tests_config/require_prettier.js"
```

Comment:
Why not `"prettier"`?

Reply:
Two reasons:
1. If moduleNameMapper is misconfogured, well be requiring the prettier from node_modules.
2. I was getting errors from jest due to something related to Haste.

Reply:
If moduleNameMapper is misconfigured, tests will fail either way, but I understand that a missing `local.js` will be more easy to know what's happening